### ISSUE TITLE
Task Loader Mapping: Namespace Logic?

### ISSUE THREAD

**Issue by valentinaschueller** [valentinaschueller/scriptengine-tasks-ecearth#2]
Tasks are related to different components of EC-Earth, e.g. computational performance, OpenIFS, Nemo. Where should this structure be visible? Idea:

```yaml
ece.mon.nemo.global_avg:
     - ...
ece.mon.oifs.global_avg:
     - ...
```

**Comment by uwefladrich** [comment]
At least we are sure about the `ece.mon.` prefix.

The rest is yet unclear, but the discussion should be connected to #3 and #44.

**Comment by valentinaschueller** [comment]
This discussion is closed by the solution mentioned in #105 - see the updated documentation on Processing Tasks/the Processing Pattern
